### PR TITLE
Add MACE aggregation model

### PR DESCRIPTION
**Pull request by pilot7747**
I have added the MACE aggregation model.
https://www.cs.cmu.edu/~hovy/papers/13HLT-MACE.pdf

## Description
Based on the original VB inference implementation, I wrote it in Python.

### Connected issues (if any)
https://github.com/Toloka/crowd-kit/issues/5

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation and examples improvement (changes affected documentation and/or examples)

## Checklist:
- [x] I have read the **CONTRIBUTING** document.
- [x] I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by dustalov — CHANGES_REQUESTED** [review]
All good. Have you tried running benchmarks from Hovy et al.? Does our code produce comparable results?

**Comment by dustalov** [comment]
Also, please add an entry to `README.md`.

**Comment by pilot7747** [comment]
@dustalov It would be great, but I wasn't able to find the original datasets because they are not available. Seems like links to ai.stanford.com have expired. However, I did compared it with the Java implementation and checked if it returns the same numbers at every step on example datasets.

File: tests/aggregation/test_mace_aggregation.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+from crowdkit.aggregation.utils import evaluate, evaluate_equal
+
+
+@pytest.mark.parametrize(
+    'method', ['vb', 'em']
+)
+def test_aggregate_mace_on_toy_ysda(
+        method: str,
+        toy_answers_df: pd.DataFrame,
+        toy_ground_truth_df: pd.DataFrame
```
[the remark this answers is not in the document]
Comment (pilot7747) [reply tests/aggregation/test_mace_aggregation.py]: Done

**Review by dustalov — APPROVED** [review]

**Pull request merged**